server/src/__tests__/router.test.ts

```diff
@@ -12,7 +12,17 @@ describe('Get endpoint', () => {
     })
 })
 
-describe('delete endpoint', () => {
+describe('Delete endpoint', () => {
+    beforeAll(async () => {
+        await request(server).post('/api/create').send({ inputUrl: 'some url' });
+    });
+    it('should delete a post input url', async () => {
+        const res = await request(server).delete('/api/delete').send({ inputUrl: 'some url' });
+        expect(res.body.message).toEqual('Url deleted');
+    })
+})
```

Comment:
Awesome!! you have added test cases as well. Would you mind updating the README.md file as well, regarding the new API?

Reply:
Yup! I'll make the requested updates!